pkg/builder/local_build_executor_test.go

```diff
@@ -33,7 +33,7 @@ func TestLocalBuildExecutorInvalidActionDigest(t *testing.T) {
 	buildDirectoryCreator := mock.NewMockBuildDirectoryCreator(ctrl)
```

Comment:
Could you add some unit testing coverage for device node creation as well?

Reply:
Do I need to somehow let the test create actual device nodes, or is it enough to simply mock the directory `<input root>/dev` and check the `Mknod` calls are made against it when there is a non-empty map of device nodes passed to `NewLocalBuildExecutor()`?

Reply:
And if it's the second case can I do something like in #56 where I just modified an existing test?

Reply:
Testing the creation of actual device nodes is likely impossible, because that would require the unit tests to run with super-user rights, which is impractical. Mocking the "dev" directory and capturing Mknod() calls is all right>

With regards to your second question: for the success case it's sufficient to just modify one of the existing tests. You likely want to test the failure case as well (when Mknod() fails). This should better go in its own test.